salesagility/SuiteCRM
# Fixed #2049 - 7.7.2 - Calendar Activities are off by 1 day

## Description



Fixes a bug with calendar events being off by 1 day.

## Motivation and Context

Issue reference: #2049

## How To Test This

1. Go to Calender module.
2. Click Week view.
3. Add a new event by clicking anywhere in the week view.
4. Check that the event is displayed correctly.

*Try in various timezones

## Types of changes


### Final checklist
